tools/wtv/.shed.yml

```diff
@@ -0,0 +1,15 @@
+name: wtv
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/wtv"
+homepage_url: "https://recetox.github.io/wtv/"

```

Comment:
remote repository url should point towards the Galaxy tool wrapper path on Github, homepage_url to the original development repository